controllers.js

```diff
@@ -55,3 +55,19 @@ exports.postCommentByArticleId = (req, res, next) => {
         })
 
 }
+
+exports.patchArticleById = (req, res, next) => {
+    updateArticleVotesById(req.params.article_id, req.body.inc_votes)
+    .then((article) => {
+        if (!article) {
```

Comment:
This should be thrown as soon as we are able to, which would be down in the model.